contrib/README.md

```diff
@@ -0,0 +1,368 @@
+    1. [Query all events](#query-all-events)
+    1. [Register a subscription](#register-a-subscription)
+    1. [Query all subscriptions](#query-all-subscriptions)
+1. [Configuration](#configuration)
+1. [Cleanup](#cleanup)
+
+## Quickstart
+
+Make sure you have helm installed on your machine and run `helm init` on your k8s cluster. This will set up the
+`helm` and `tiller` functions required for easy deployment of config files to your cluster. You can follow
+instructions [here](https://docs.helm.sh/using_helm/#quickstart) if you have not set this up previously.
+
+**NOTE:** This portion of the config expects you to have a pre-existing kubernetes cluster (not minikube). For 
+local development please check the [minikube](MINIKUBE.md) information below.
+
+Once installed, navigate to the `event-gateway/contrib/helm` folder and install the following components:
+
+**etcd-operator**
+```bash
+helm install stable/etcd-operator --name ego [--namespace <namespace>]
+```
+
+**event-gateway**
+```bash
+helm install event-gateway --name eg [--namespace <namespace>]
+```
+
+This will install each of the `etcd-operator` and `event-gateway` into the `default` namespace in kubernetes. Please note,
+this namespace has no bearing on your Event Gateway `spaces` as outlined in the [docs](https://github.com/serverless/event-gateway/blob/master/README.md). If you'd like
+to install `etcd-operator` and `event-gateway` in another namespace, add the `--namespace <namespace>` option to
+ both `helm install` commands above.
+
+Next we'll need to collect the Event Gateway IP to use on the CLI. We have a couple of options available to reference the
+internal services of the kubernetes cluster exposed via Ingress:
+
+1. add Ingress IP to /etc/hosts (recommended)
+
+   This method enables us to reference the `event-gateway` from the hostname we configured in the Ingress module. This document 
+   assumes the name to be `eventgateway.minikube` so please update the instructions to your naming convention if you need.
+  
+   ```bash
+     echo "$(kubectl get ingress event-gateway-ingress -o jsonpath={.status.loadBalancer.ingress[0].ip}) eventgateway.minikube" | sudo tee -a "/etc/hosts"
+   ```
+
+1. use Ingress IP and pass header to request
+
+   With this method we access the `event-gateway` using the IP of the Ingress directly. Since the Ingress was configured to 
+   receive all connections from the `eventgateway.minikube` host, you'll need to pass this as a header value to the request.
+   
+   ```bash
+   export EVENT_GATEWAY_URL=$(kubectl get ingress event-gateway-ingress -o jsonpath={.status.loadBalancer.ingress[0].ip})
+   curl --request GET \
+        --url http://{EVENT_GATEWAY_URL}/v1/metrics \
+        --header 'content-type: application/json' \
+        --header 'host: eventgateway.minikube'
+   ```
+
+With your environment set up, you can now jump to the [examples](#examples) section to put your `event-gateway` to use!
+
+### Using helm
```

Comment:
Is this `Using helm` section a duplicate of the section above? Should I be running both the commands above and the commands in this `Using helm` section?

Reply:
Agree that this section is confusing and seems redundant. Can we simply remove this and rename Quickstart as "Quickstart using helm". We can add a CRD Quickstart later